ha/setup/validate_cluster.py

```diff
@@ -0,0 +1,24 @@
+import argparse 
+import sys
+from typing import Any
```

Comment:
Do we need separate cli for test or is it only for testing.

If only for integration related test then can we move it in test/integration/setup/

Reply:
I would like to clarify this requirement with @ajaysrivas . 
At this moment it looks reasonable to have CLI tools to run some functions on real cluster.